system-addon/content-src/activity-stream-windows.scss

```diff
@@ -0,0 +1,7 @@
+/* This is the windows variant */
```

Comment:
Just to be explicit, this comment *block* will appear in the generated file. Which I suppose would be handy for debugging?

Reply:
Yep, that's the idea